web/packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -112,10 +112,15 @@ export function DesktopSession(props: State) {
     }
     const open = errorText !== '';
 
-    return { open, text: errorText };
+    return {
+      open,
+      text: errorText,
+      isError: unknownConnectionError || errorText === 'RDP connection failed',
```

Comment:
I agree that hardcoding is the way to go for now, however a problem I see with hardcoding this value is that for any other value of `errorText` besides `'RDP connection failed'` we will also show an `Info` rather than a `Danger`. I think it's less error prone to just hardcode the one specific message we want to display in an `Info`.

Reply:
There's not one specific message we want to display in an info. 

Any time the server sends us a disconnect reason, we consider it an info since it was an "expected" disconnect. 

Unfortunately, the way this is written makes it really difficult to tell whether or not we received a disconnect from the server. 

Let's not split hairs here. This addresses a costumer concern and is a clear improvement over always showing a big red danger box.